firebase-perf/src/main/java/com/google/firebase/perf/metrics/AppStartTrace.java

```diff
@@ -227,11 +272,24 @@ public synchronized void onActivityResumed(Activity activity) {
 
     // Log the app start trace in a non-main thread.
     executorService.execute(this::logAppStartTrace);
+  }
 
-    if (isRegisteredForLifecycleCallbacks) {
-      // After AppStart trace is logged, we can unregister this callback.
-      unregisterActivityLifecycleCallbacks();
-    }
+  private void logColdStart(Timer start, Timer end, PerfSession session) {
+    TraceMetric.Builder metric =
+        TraceMetric.newBuilder()
+            .setName("_experiment_as_ttid")
+            .setClientStartTimeUs(start.getMicros())
+            .setDurationUs(start.getDurationMicros(end));
+
+    TraceMetric.Builder subtrace =
+        TraceMetric.newBuilder()
+            .setName("_experiment_classLoadTime")
```

Comment:
What's the purpose of this sub trace?

Reply:
This captures the old-start, which is class-load of `FirebasePerfProvider.class`. I just want to have more data to study the difference that the new start would have.

Reply:
sg